plugins/pom-analyzer/src/test/java/eu/f4sten/pomanalyzer/utils/MavenRepositoryUtilsTest.java

```diff
@@ -109,6 +112,11 @@ public void nonExistingPom() {
             var a = newResolutionResult(SOME_COORD, ARTIFACT_REPO, f);
             sut.downloadPomToTemp(a);
         });
+        var isFlakyCrash = IS_OS_WINDOWS && e.getCause() instanceof ConnectException;
+        if (isFlakyCrash) {
+            System.err.println("[WARN] Flaky test on Windows was unable to connect");
```

Comment:
😄

Reply:
I could not reproduce the test errors on a Windows machine, so unless the GitHub Windows runners become more deterministic, I see no other way to prevent the flakiness of this test.